CustomPermissionClaimHandler/src/main/java/org/wso2/custom/claim/internal/PermissionClaimHandlingComponent.java

```diff
@@ -0,0 +1,57 @@
+package org.wso2.custom.claim.internal;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.osgi.service.component.ComponentContext;
+import org.wso2.carbon.user.core.service.RealmService;
+
+/**
+ * @scr.component name="custom.permission.claim.handler"
+ * immediate="true"
+ * @scr.reference name="user.realmservice.default"
+ * interface="org.wso2.carbon.user.core.service.RealmService"
+ * cardinality="1..1" policy="dynamic" bind="setRealmService" unbind="unsetRealmService"
+ */
+public class PermissionClaimHandlingComponent {
+
+    private static Log log = LogFactory.getLog(PermissionClaimHandlingComponent.class);
+    private static RealmService realmService;
+
+    protected void activate(ComponentContext ctxt) {
+
+        try {
```

Comment:
I think this try-catch block is pointless. Can we just add a debug log? Please correct me if I'm wrong.